python/tvm/script/builder/tir/for_frame.py

```diff
@@ -15,42 +15,44 @@
 # specific language governing permissions and limitations
 # under the License.
 """TVM Script TIR For Frame"""
-from tvm._ffi import register_object as _register_object
+from typing import List
 
+from tvm._ffi import register_object as _register_object
 from tvm.tir import Var
 
 from . import _ffi_api
-from ._ffi_api import _base_ffi_api
+from .. import _ffi_api as _base_ffi_api
 from .base import TIRFrame
-from typing import List
 
 
 @_register_object("script.builder.tir.ForFrame")
 class ForFrame(TIRFrame):
     def __enter__(self) -> List[Var]:
-        _base_ffi_api.FrameEnter(self)
+        _base_ffi_api.FrameEnter(self)  # pylint: disable=no-member # type: ignore
         return self.vars
 
 
-def serial(min_val, extent, attrs) -> ForFrame:
-    return _ffi_api.Serial(min_val, extent, attrs)
+def serial(start, stop, annotations) -> ForFrame:
```

Comment:
@cyx-6 Usually we make sure that the names of arguments are consistent with that in C++ (roughly)

Reply:
Got it.